### ISSUE TITLE
[fr] user feedback "tous" vs "tout"

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#6211]
User says:
> nul ne détecte pas la faute tous d'abord qui prend un t et pas un s

**Comment by udomai** [comment]
@Rebecca-auque, the user is being quite rude, but they have a point. This might be a quick win.

> J'aimerais **tous** d'abord vous remercier.
> Nous aimerions **tous** d'abord vous remercier.

Could a sentence like the second one actually be meant as "We all would like to thank you first" instead of "At the very beginning, we would like to thank you"? Or could we match any occurrence of "tous d'abord"? Are there any other (semi-)fixed expressions with "tout" where we could spot a "tous"?

**Comment by Rebecca-auque** [comment]
Yes, it could be 'We all would like to thank you first' but it is unlikely, and the structure would ambiguous. In that case, it could be modified into 'First of all, we all would like to thank you'. Another unlikely case would be that the user actually misplaced the adverb, so it could become 'First of all, we would like to thank you all.'   The most likely case would be that 'tous' is actually 'tout'. Yes, there are lots of semi-fixed expression with 'tout'. I'll create rules for it.
